common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -99,7 +99,16 @@
   HDFS_FILE_NOT_FOUND("File {0} with id {1} is not found"),
 
   // file system master
+  FILE_MUST_HAVE_VALID_PARENT("{0} does not have a valid parent"),
```

Comment:
do non-root files ever not have valid parents? When would you expect the user to see this?